src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -3223,6 +3231,13 @@ bool AutomationInstrumentClipView::isOnParameterGridMenuView() {
 //displays patched param names or midi cc names
 void AutomationInstrumentClipView::displayParameterName(int32_t paramID) {
 
```

Comment:
when drawing the parameter name, check that a pad is not currently being pressed and that the last pad selected knob's position was recorded

if a pad is pressed and knob position recorded, display parameter value popup instead of drawing the parameter name and exit the function